File: test-artifacts/p-visitor-py/function-invocation-replacer/int88888888_003_out.py
def f_gold(arr, l, r, x):
    if r >= l:
        mid = l + (r - l) // 2
        if arr[mid] == x:
            return mid
        elif arr[mid] > x:
            return 88888888
        else:
            return 88888888
    else:
        return -1

File: test-artifacts/p-visitor-js/function-invocation-replacer/int88888888_003_out.js
function f_gold(arr, l, r, x) {
    if (r >= l) {
        const mid = l + Math.floor((r - l) / 2);
        if (arr[mid] === x) {
            return mid;
        } else if (arr[mid] > x) {
            return 88888888;
        } else {
            return 88888888;
        }
    } else {
        return -1;
    }
}